modules/wordads/wordads.php

```diff
@@ -319,24 +322,30 @@ function get_ad( $spot, $type = 'iponweb' ) {
 		$snippet = '';
 		$blocker_unit = 'mrec';
 		if ( 'iponweb' == $type ) {
-			$section_id = WORDADS_API_TEST_ID;
+			// Default to mrec
 			$width = 300;
 			$height = 250;
+
+			$section_id = WORDADS_API_TEST_ID;
 			$second_belowpost = '';
 			$snippet = '';
 			if ( 'top' == $spot ) {
-				// mrec for mobile, leaderboard for desktop
-				$section_id = 0 === $this->params->blog_id ? WORDADS_API_TEST_ID : $this->params->blog_id . '2';
-				$width = $this->params->mobile_device ? 300 : 728;
-				$height = $this->params->mobile_device ? 250 : 90;
-				$blocker_unit = $this->params->mobile_device ? 'top_mrec' : 'top';
-				$snippet = $this->get_ad_snippet( $section_id, $height, $width, $blocker_unit );
+				// Switch to leaderboard on desktop
+				if ( ! $this->params->mobile_device ) {
+					$width = 728;
+					$height = 90;
+				}
+
+				$section_id     = 0 === $this->params->blog_id  ? WORDADS_API_TEST_ID   : $this->params->blog_id . '2';
+				$blocker_unit   = $this->params->mobile_device  ? 'top_mrec'            : 'top';
+				$snippet        = $this->get_ad_snippet( $section_id, $height, $width, $blocker_unit );
+
 			} else if ( 'belowpost' == $spot ) {
-				$section_id = 0 === $this->params->blog_id ? WORDADS_API_TEST_ID : $this->params->blog_id . '1';
+				$section_id     = 0 === $this->params->blog_id	? WORDADS_API_TEST_ID   : $this->params->blog_id . '1';
+				$snippet = $this->get_ad_snippet( $section_id, $height, $width, 'mrec', 'float:left;margin-right:5px;margin-top:0px;' );
 				$width = 300;
 				$height = 250;
 
-				$snippet = $this->get_ad_snippet( $section_id, $height, $width, 'mrec', 'float:left;margin-right:5px;margin-top:0px;' );
```

Comment:
Do I understand correctly that after moving this line up we're going to enable two different dimensions for ads below content if `wordads_second_belowpost` is true? Or is setting width and height here in this condition branch redundant?